redwood-cli/src/main/kotlin/app/cash/redwood/generator/composeGeneration.kt

```diff
@@ -164,7 +165,7 @@ internal fun generateComposable(
           }
 
           val updateLambda = CodeBlock.builder()
-            .add("set(layoutModifier) { layoutModifiers = layoutModifier }\n")
```

Comment:
Huh I didn't realize your other change to switch this to `it` was intentional and not a typo. I was not aware that the value was also supplied to this lambda (although I guess it makes sense since the function references work).

Reply:
Yep, I could update it to use `it` again for consistency with the other properties. That would make it non-capturing, right?

Reply:
Yeah let's update to non-capturing. I'm actually surprised it worked at all in the capturing form. Maybe a pre-1.0 thing that got changed and I never tracked it. I was pretty sure that I copied their examples from Compose UI but maybe I just got it wrong.